microsoftgraph/microsoft-graph-explorer-v4
# Fix: Restore Resources search box functionality

## Overview

Fixes #2676 
Removes the documentation component that was causing the search box to fail.
The previous change was causing a regression error

### Demo

https://github.com/microsoftgraph/microsoft-graph-explorer-v4/assets/18407044/eb6b917a-d4c4-48f7-aeff-3ff7cbd7a611

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Check out this branch
* Load graph Explorer locally
* Click on **Resources** tab
* Select the search box and search for a resource e.g messages
* Observe page doesn't hang
